backend/core/main.py

```diff
@@ -1,4 +1,6 @@
 import os
+from dotenv import load_dotenv
+load_dotenv()
```

Comment:
We already use Loadenv kind of with BaseSettings from Pydantic


https://docs.pydantic.dev/latest/usage/pydantic_settings/

Reply:
Can you put that in the 

if __name__ == "__main__":

So it doesn't disrupt anything just in case ;)

Reply:
Yes - I still need it at the top of the file - but I will also wrap it in name==main so that it doesn't execute otherwise